changelog.d/7263.bugfix

```diff
@@ -0,0 +1 @@
+Create user with admin API if MAU limit is reached.
```

Comment:
this sounds like every time the MAU limit is reached, we will create a new user!

```suggestion
Allow new users to be registered via the admin API even if the monthly active user limit has been reached.
```